src/ralph/backends/data/base.py

```diff
@@ -79,24 +116,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-class BaseDataBackend(ABC):
+class BaseDataBackend(BaseBackend):
     """Base data backend interface."""
 
     type = "data"
     name = "base"
+    logger = logger
+    query_class = BaseQuery  # we rename query_model to query_class
     query_model = BaseQuery
-    default_operation_type = BaseOperationType.INDEX
+    default_operation_type: BaseOperationType = BaseOperationType.INDEX
+    unsupported_operation_types: Set[BaseOperationType] = set()
     settings_class = BaseDataBackendSettings
+    settings: settings_class
 
-    @abstractmethod
-    def __init__(self, settings: Union[settings_class, None] = None):
-        """Instantiate the data backend.
-
-        Args:
-            settings (BaseDataBackendSettings or None): The data backend settings.
-                If `settings` is `None`, a default settings instance is used instead.
-        """
-
+    # This method is replaced by validate_backend_query
     def validate_query(
         self, query: Union[str, dict, BaseQuery, None] = None
     ) -> BaseQuery:
```

Comment:
So the intention is to remove this method ?

Reply:
Yes) partially) we move it out of the class, add two new arguments to it (query_class and logger) and rename it)